app/javascript/tooltips.js

```diff
@@ -1,10 +1,14 @@
 import tippy from "tippy.js"
 
-document.addEventListener('turbolinks:load', function() {
+document.addEventListener('turbolinks:load', applyTips)
+
+function applyTips() {
   tippy('.gold-split', {placement: 'left'})
   tippy('.tip', {placement: 'top'})
   tippy('.tip-top', {placement: 'top'})
   tippy('.tip-bottom', {placement: 'bottom'})
   tippy('.tip-right', {placement: 'right'})
   tippy('.tip-left', {placement: 'left'})
-})
+}
+
+export { applyTips }
```

Comment:
what was the reason for this change?

Reply:
To apply the tips when we receive new html from ActionCable (user badges mostly).